contrib/cirrus/setup_environment.sh

```diff
@@ -272,6 +272,11 @@ case "$TEST_FLAVOR" in
         ;;
     build) make clean ;;
     unit) ;;
+    compose_v2)
+        dnf -y remove docker-compose
+        curl -SL https://github.com/docker/compose/releases/download/v2.2.3/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
```

Comment:
I'm always leery to have a specified version in tests rather than latest.
@edsantiago probably has a neater way, but you could get/set the version from GitHub into an envar and then use it:

```
# export VERSION=`curl -s https://github.com/docker/compose/releases/latest | grep "https://github.com/docker/compose/releases/tag/*"  | sed 's/^.*tag\///' | sed 's/\">.*//'`
#  echo $VERSION
v2.3.3
# curl -SL https://github.com/docker/compose/releases/download/${VERSION}/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
```

Reply:
The latest version can break at any time. Using a fixed version is a good idea IMO.
I think we should download the compose binary at image build time to prevent unnecessary flakes. We could put it at a fixed place and then just mv it here to /usr/bin

Reply:
Just FYI, if you do end up deciding to download the latest version using the URL `https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64` is much easier (and more robust) than `grep`ing and `sed`ing through GitHub's HTML :)